README.md

```diff
@@ -150,10 +148,73 @@ let scaleString = scale.stringValue
 let scaleInt = scale.integerValue
 ```
 
+#### Type Declarations ####
+
+When declaring types, the colon should be placed immediately after the identifier followed by one space
+and the type name.
+
+```swift
+
+var intValue: Int
+
+// Do NOT do this
+var intValue : Int
+
+```
+
+In all use-cases, the colon should be associated with the left-most item with no spaces preceding and one space afterwards:
+
+```swift
+let myDictionary: [String: AnyObject] = ["String": 0]
+```
+
+#### typealias ####
+
+Typealias declaration should precede any other type declaration.
+
+```swift
+// typealias ClosureType = (ParameterTypes) -> (ReturnType)
+typealias AgeAndNameProcessor = (Integer, String) -> Void
+
+var intValue: Int
+
+class Object {
+	private var someString = ""
+
+	func returnOne() {
+		return 1
+	}
+}
+```
+
```

Comment:
re:Chris comment: `If declaring a typealias for protocol conformance, it should be declared at the top of the class, before anything else`